vms/volunteer/views.py

```diff
@@ -98,7 +100,7 @@ def form_valid(self, form):
             else:
                 return render(self.request, 'volunteer/edit.html',
                               {'form': form, 'organization_list': organization_list, 'volunteer': volunteer,
-                               'resume_invalid': True,})
+                               'resume_invalid': True})

```

Comment:
👋 is this autopep8, also above new lines?

Reply:
Fixed this issue and all the spacing problems.